tests/baselines/reference/declarationEmitLocalClassDeclarationMixin.js

```diff
@@ -107,3 +107,4 @@ declare const FilteredThing_base: {
 export declare class FilteredThing extends FilteredThing_base {
     match(path: string): boolean;
 }
+export {};

```

Comment:
where is this coming from ?

Reply:
```
declare const FilteredThing_base: {
     new (...args: any[]): {	     new (...args: any[]): {
         match(path: string): boolean;	         match(path: string): boolean;
         thing: number;	         thing: number;
     };	     };
 } & typeof Unmixed;	 } & typeof Unmixed;
```
doesn't exist at runtime (we just emit it so we have a thing for the `extends` clause) - this scopes the file so it's actually private.